agent/utils/fips_linux_test.go

```diff
@@ -0,0 +1,52 @@
+//go:build !windows && unit
```

Comment:
nit: this build constraint should probably be `linux` so that it doesn't include macos or others